lib/active_merchant/billing/gateways/adyen_checkout.rb

```diff
@@ -69,6 +72,12 @@ def store(credit_card, options={})
         end
       end
 
+      def details(options)
```

Comment:
We probably should have some spec for that.

Reply:
We have tests in Chargify that test the whole flow with 3DS. Writing tests here for 3d secure flow is very hard, so probably we would have to stub everything and I don't think that this is the best approach.

Reply:
@himn1 you'll probably be the one approving the release so you could as well tell us now what our policy is. How do we even treat AM? Is it our own gem at this point? Or are w planning on somehow joining the main fork at some point in the future?

Reply:
I've added some unit tests

Reply:
Answered in-slack
recap:
Definitely we have to make an effort soon and try to be upstream.
Step after step with a proper plan.